backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/ObjCInterop.kt

```diff
@@ -61,6 +63,9 @@ fun IrClass.isExternalObjCClass(): Boolean = this.isObjCClass() &&
             it.descriptor.annotations.hasAnnotation(externalObjCClassFqName)
         }
 
+fun ClassDescriptor.isObjCForwardDeclaration(): Boolean =
+        this.findPackage().fqName.startsWith(objcnamesForwardDeclarationsPackageName)
```

Comment:
Maybe check that it starts with `objcnames.`.

Reply:
This is exactly what is done here.

Reply:
`FqName.startsWith()` takes dots into account properly.